api/datastore/sql/migrations/migrations.go

```diff
@@ -55,6 +55,35 @@ var initialTables = [...]string{`CREATE TABLE IF NOT EXISTS routes (
 );`,
 }
 
+func checkOldMigrationTableVersionIfExists(db *sql.DB) (version int64, dirty bool, err error) {
+	migrationsTable := "schema_migrations"
+	ctx := context.Background()
+
+	// deleting old table after all
+	defer func() {
+		_, _ = db.ExecContext(ctx, "DROP TABLE IF EXISTS "+migrationsTable)
+	}()
+
+	err = db.QueryRowContext(
+		ctx, "SELECT version, dirty FROM "+migrationsTable+" LIMIT 1").
+		Scan(&version, &dirty)
+	switch {
+	case err == sql.ErrNoRows:
+		return -1, false, nil
+
+	case err != nil:
+		if e, ok := err.(*mysql.MySQLError); ok {
```

Comment:
sqlite3 is probably the most popular option (also pg, less popular)? not sure if required, but it seems like as long as we check the `dirty` bit, we can skip this error type checking altogether and just remove this, but i didn't test.

Reply:
fixed